terraform-provider-intro/order-update.md

```diff
@@ -0,0 +1,79 @@
+Change your order by updating the order resource through Terraform. In your `main.tf`, update the coffee quantity in `hashicups_order.edu` block.
+
+Change the first coffee item from `2` to `3` and change the second coffee item from `2` to `1`.
+
+```
+resource "hashicups_order" "edu" {
+  items {
+    coffee {
+      id = 3
+    }
+    quantity = 3
+  }
+  items {
+    coffee {
+      id = 2
+    }
+    quantity = 1
+  }
+}
+```
+
+Run `terraform apply` to update the order. Notice how the execution plan reflects the order change. Terraform is able to determine that these changes could be done in place rather than destroying the current order and creating a new one. This is because the modified property ([quantity](https://github.com/hashicorp/terraform-provider-hashicups/blob/master/hashicups/resource_order.go#L63)) does not have `ForceNew` set to `true`.
```

Comment:
```suggestion
Apply the order update.
```

Reply:
I'm going to keep the original one to provide more context to the command. This is supposed to serve as an "intro to providers" lab/tutorial, so explaining how the resource updates (and why it doesn't get destroyed and recreated) is important